react/locales/en.json

```diff
@@ -0,0 +1,14 @@
+{
+  "UI": {
+    "React": {
+      "hooks": {
+        "useConfirmExit": {
+          "back": "Back",
+          "leave": "Leave",
+          "title": "Do you want to leave?",
+          "message": "Some changes are not saved yet. Do you really want to leave and loose these changes?"
+        }
+      }
+    }
+  }
+}
```

Comment:
ATM we are storing the locales in the component's folder (ex: react/viewer/locales, react/AppSection/locales). Personally I find it easier to maintain

Reply:
@edas Does it miss the commit to move the files to the right folder? Or do you want to keep this structure?